integration-tests/docker/Dockerfile

```diff
@@ -25,6 +25,11 @@ RUN find /var/lib/mysql -type f -exec touch {} \; && /etc/init.d/mysql start \
 # Add Druid jars
 ADD lib/* /usr/local/druid/lib/
 
+# Download the MySQL Java connector
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y --no-install-recommends apt-utils libmysql-java
+RUN ln -sf /usr/share/java/mysql-connector-java.jar /usr/local/druid/lib/mysql-connector-java.jar
```

Comment:
I'm not sure if this is cool or not, but it seems maybe cool, on the grounds that we are not distributing the integration test docker image containing the mysql jar (it is a transient artifact generated during the running of the integration tests). I say we go with it for now at least, and focus on trying to make sure the release artifacts are done properly. In the long run we might want to switch the ITs to use derby or postgresql to avoid any whiff of complication.